akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -74,6 +74,23 @@ sealed trait HttpEntity extends jm.HttpEntity {
    */
   def withContentType(contentType: ContentType): HttpEntity
 
+  /**
+   * Apply the given size limit to this entity by returning a new entity instance which automatically verifies that the
+   * data stream encapsulated by this instance produces at most ``maxBytes`` data bytes. In case this verification fails
+   * the respective stream will be terminated with an ``EntityStreamException`` either directly at materialization
+   * time (if the Content-Length is known) or whenever more data bytes than allowed have been read.
+   *
+   * When called on ``Strict`` entities the method will return the entity itself if the length is within the bound,
```

Comment:
Right, I always mix up all those markup styles.
These tools should simply "do the right thing" ... :)